### PR TITLE
New version: InfiniteInstant.KoduGameLab version 1.6.12

### PR DESCRIPTION
**Pull request by mdanish-kh**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65538)

### CODE REVIEW

File: manifests/i/InfiniteInstant/KoduGameLab/1.6.12.0/InfiniteInstant.KoduGameLab.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+PackageLocale: en-US
+Publisher: InfiniteInstant
+PublisherUrl: http://www.infiniteinstant.com/
+# PublisherSupportUrl: 
+PrivacyUrl: https://www.kodugamelab.com/privacy/
+# Author: 
+PackageName: Kodu
+# PackageUrl: 
+License: MIT License
+LicenseUrl: https://github.com/scoy/KoduGameLab/blob/main/LICENSE.TXT
```
Comment (Trenly) [inline manifests/i/InfiniteInstant/KoduGameLab/1.6.12.0/InfiniteInstant.KoduGameLab.locale.en-US.yaml:15]:
```suggestion
License: Proprietary
LicenseUrl: https://www.kodugamelab.com/EULA
```
License for the code is not the same as the package License

File: manifests/i/InfiniteInstant/KoduGameLab/1.6.12.0/InfiniteInstant.KoduGameLab.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: InfiniteInstant.KoduGameLab
+PackageVersion: 1.6.12.0
```
Comment (Trenly) [inline manifests/i/InfiniteInstant/KoduGameLab/1.6.12.0/InfiniteInstant.KoduGameLab.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.6.12
```

File: manifests/i/InfiniteInstant/KoduGameLab/1.6.12.0/InfiniteInstant.KoduGameLab.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: InfiniteInstant.KoduGameLab
+PackageVersion: 1.6.12.0
```
Comment (Trenly) [inline manifests/i/InfiniteInstant/KoduGameLab/1.6.12.0/InfiniteInstant.KoduGameLab.installer.yaml:5]:
```suggestion
PackageVersion: 1.6.12
```

File: manifests/i/InfiniteInstant/KoduGameLab/1.6.12.0/InfiniteInstant.KoduGameLab.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: InfiniteInstant.KoduGameLab
+PackageVersion: 1.6.12.0
```
Comment (Trenly) [inline manifests/i/InfiniteInstant/KoduGameLab/1.6.12.0/InfiniteInstant.KoduGameLab.yaml:5]:
```suggestion
PackageVersion: 1.6.12
```

File: manifests/i/InfiniteInstant/KoduGameLab/1.6.12.0/InfiniteInstant.KoduGameLab.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+PackageLocale: en-US
+Publisher: InfiniteInstant
+PublisherUrl: http://www.infiniteinstant.com/
+# PublisherSupportUrl: 
+PrivacyUrl: https://www.kodugamelab.com/privacy/
+# Author: 
+PackageName: Kodu
```
Comment (Trenly) [inline manifests/i/InfiniteInstant/KoduGameLab/1.6.12.0/InfiniteInstant.KoduGameLab.locale.en-US.yaml:12]:
```suggestion
PackageName: Kodu Game Lab
```
Must match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]
The PackageVersion must match control panel; When updating this, you will need to update the name of the version folder also. You can do this in GitHub web by pressing <kbd>.</kbd>

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**